src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -149,6 +156,17 @@ We support
             }
             else
             {
+                string rawRequirementType = configurationSection[RequirementTypeKeyword];
+
+                //
+                // If requirement type is specified, parse it and set the requirementType variable
+                if (!string.IsNullOrEmpty(rawRequirementType) && !Enum.TryParse(rawRequirementType, true, out requirementType))
```

Comment:
`Enum.TryParse(rawRequirementType, true, out requirementType)`,

For boolean parameters where we pass a boolean literal, I'd recommend to use parameter name to help the reader know what that flag is for.

Like `Enum.TryParse(rawRequirementType, ignoreCase: true, out requirementType)`,